trillian/ctfe/config.go

```diff
@@ -167,6 +173,24 @@ func ValidateLogConfig(cfg *configpb.LogConfig) (*ValidatedLogConfig, error) {
 		return nil, errors.New("expected merge delay exceeds MMD")
 	}
 
+	if sth := cfg.FrozenSth; sth != nil {
+		verifier, err := ct.NewSignatureVerifier(vCfg.PubKey)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create signature verifier: %v", err)
+		}
+		if vCfg.FrozenSTH, err = (&ct.GetSTHResponse{
```

Comment:
Maybe that method shouldn't have had a pointer receiver :/

Reply:
Not untrue.